__tests__/park.test.js

```diff
@@ -20,109 +20,86 @@ async function setupDb() {
     convertEmptyValues: true
   });
 
-  for(const park of PARKSLIST) {
-    await docClient
-      .put({
-        TableName: TABLE_NAME,
-        Item: park
-      })
-      .promise();
```

Comment:
Here and below are some of the park POST/GET endpoint code that should be reinstated.

Reply:
Reinstated. Thanks for pointing it out.